experimental/Matrix/matrix.jl

```diff
@@ -67,7 +67,7 @@ function MatrixGroup(matrices::Vector{<:MatrixElem{T}}) where T <: Union{fmpz, f
             if !isinvertible(mat)
                 error("At least one matrix is not invertible.")
             end
-            if size(mat) != (n, n)
+            if Oscar.size(mat) != (n, n)
                 error("At least one matrix is not quadratic or not the same size.")
```

Comment:
I've been told that it is called "square matrix" and not "quadratic matrix" in English.

Reply:
Indeed. Also note that the function `issquare` can be used for this purpose.